pos_receipt_hide_price/static/src/js/OrderReceipt.js

```diff
@@ -0,0 +1,19 @@
+odoo.define("pos_receipt_hide_price.OrderReceipt", function (require) {
+    "use strict";
+
+    const OrderReceipt = require("point_of_sale.OrderReceipt");
+    const Registries = require("point_of_sale.Registries");
+
+    const HidePriceOrderReceipt = (OrderReceipt) =>
+        class extends OrderReceipt {
+            constructor(_, {hpState}) {
```

Comment:
If I understand correctly hp is for hidePrice ? I think ``hidePriceState`` could be more explicit. don't you think ?

Reply:
Agreed